core/domain/topic_jobs_one_off.py

```diff
@@ -96,6 +98,35 @@ def reduce(key, values):
             yield (key, values)
 
 
+class InteractionsInATopicAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """An audit job to report all interaction ids used in a Topic."""
```

Comment:
Add a newline explaining that the job is just needed for February release.

Reply:
I think we might not need to merge this PR, we would just run it on the backup server. It is only to finalise some requirements for the Android project. I'll add to the description.